deploy.sh

```diff
@@ -831,13 +842,14 @@ function variables {
   fi
 
   if [ -s "./pull-secret.txt" ]; then
-    log "Found pull-secret.txt in current directory"
     cp -f pull-secret.txt ./"$ARTIFACTS_DIR"/data/
+    echo "pull_secret_file = \"$PWD/$ARTIFACTS_DIR/data/pull-secret.txt\"" >> $VAR_TEMPLATE
```

Comment:
Now I see it I think it's better not to copy and use it from CWD. Set the pull secret file var to PWD